js-applet/src/index.ts

```diff
@@ -38,6 +38,14 @@ class PyNVL {
     if (tooltip !== null) {
       this.hoverInteraction = new HoverInteraction(this.nvl)
 
+      const truncateValue = (value: any, maxLength: number = 100): string => {
```

Comment:
Not sure this maxLength will be good for all window sizes, but let's not worry about it for now